### PR TITLE
Move to new logs

### PR DESCRIPTION
**Pull request by yarovyii**
Signed-off-by: Hlib Yarovyi <HlibYarovyi@eaton.com>

### CODE REVIEW

File: src/osrv.h [hunk 1]
```diff
@@ -88,19 +88,19 @@ inline int s_osrv_load(s_osrv_t* self)
...
     if (!self->state_file) {
-        log_warning("There is no state path set-up, can't load the state");
+        logDebug("There is no state path set-up, can't load the state");
         return -1;
     }
 
     zconfig_t* root = zconfig_load(self->state_file);
     if (!root) {
-        log_error("Can't load configuration from %s: %m", self->state_file);
+        logError("Can't load configuration from {}: %m", self->state_file);
```
Comment (etnKlendathu) [inline src/osrv.h:97]: What is %m?

**Review by etnKlendathu — APPROVED** [review]
What could be wrong? :)

File: src/osrv.h (hunk 1 above)
Comment (yarovyii) [reply src/osrv.h:97]: no idea

Comment (etnKlendathu) [reply src/osrv.h:97]: Bad answer :), maybe better to remove it?
Below same log without it.

Comment (perrettecl) [reply src/osrv.h:97]: It could be probably where the segfault come from...

File: src/data.cc [hunk 2]
```diff
@@ -265,11 +263,12 @@ zlistx_t* data_get_dead(data_t* self)
     zlistx_t* dead = zlistx_new();
 
     uint64_t now_sec = uint64_t(zclock_time() / 1000);
-    log_debug("now=%" PRIu64 "s", now_sec);
+    logDebug("now={}s", now_sec);
     for (expiration_t* e = reinterpret_cast<expiration_t*>(zhashx_first(self->assets)); e != NULL;
          e               = reinterpret_cast<expiration_t*>(zhashx_next(self->assets))) {
         void* asset_name = const_cast<void*>(zhashx_cursor(self->assets));
-        log_debug("asset: name=%s, ttl=%" PRIu64 ", expires_at=%" PRIu64, asset_name, e->ttl_sec, expiration_get(e));
+        logDebug("asset: name={}, ttl={}, expires_at={}", static_cast<char*>(asset_name), e->ttl_sec,
```
Comment (etnKlendathu) [inline src/data.cc:270]: why static_cast here? this produce a error.

Comment (yarovyii) [reply src/data.cc:270]: because asset_name is void* and logDebug print out address

File: src/fty-outage.cc [hunk 3]
```diff
@@ -24,93 +24,86 @@
...
-        if (argn < argc - 1) param = argv [argn+1];
-
-        if (streq (argv [argn], "--help")
-        ||  streq (argv [argn], "-h")) {
-            puts ("fty-outage [options] ...");
-            puts ("  --verbose / -v         verbose test output");
-            puts ("  --help / -h            this information");
-            puts ("  -c|--config            path to config file");
+        char* param = NULL;
+        if (argn < argc - 1)
```
Comment (perrettecl) [inline src/fty-outage.cc]: put bracket arround the if, it's safer.

**Review by perrettecl — CHANGES_REQUESTED** [review]

File: src/fty-outage-server.cc [hunk 4]
```diff
@@ -367,12 +366,12 @@ static void fty_outage_handle_mailbox(s_osrv_t* self, zmsg_t** msg)
     if (msg && *msg) {
         char* message_type = zmsg_popstr(*msg);
         if (!message_type) {
-            log_warning("Expected message of type REQUEST");
+            logDebug("Expected message of type REQUEST");
```
Comment (ericclappier-eaton) [inline src/fty-outage-server.cc]: Warning ?

File: src/fty-outage-server.cc [hunk 5]
```diff
@@ -367,12 +366,12 @@ static void fty_outage_handle_mailbox(s_osrv_t* self, zmsg_t** msg)
...
             return;
         }
         char* zuuid = zmsg_popstr(*msg);
         if (!zuuid) {
-            log_warning("Expected zuuid");
+            logDebug("Expected zuuid");
```
Comment (ericclappier-eaton) [inline src/fty-outage-server.cc]: Warning ?

File: src/fty-outage-server.cc [hunk 6]
```diff
@@ -387,7 +386,7 @@ static void fty_outage_handle_mailbox(s_osrv_t* self, zmsg_t** msg)
 
         if (streq(message_type, "REQUEST")) {
             if (!command) {
-                log_warning("Expected command");
+                logDebug("Expected command");
```
Comment (ericclappier-eaton) [inline src/fty-outage-server.cc]: Warning ?

File: src/fty-outage-server.cc [hunk 7]
```diff
@@ -455,13 +454,13 @@ static void fty_outage_handle_mailbox(s_osrv_t* self, zmsg_t** msg)
...
                 // command is not expected
-                log_warning("'%s': invalid command", command);
+                logDebug("'{}': invalid command", command);
                 zmsg_addstr(reply, "ERROR");
                 zmsg_addstr(reply, "Invalid command");
             }
         } else {
             // message_type is not expected
-            log_warning("'%s': invalid message type", message_type);
+            logDebug("'{}': invalid message type", message_type);
```
Comment (ericclappier-eaton) [inline src/fty-outage-server.cc]: Warning ?

File: src/osrv.h [hunk 8]
```diff
@@ -59,7 +59,7 @@ inline int s_osrv_save(s_osrv_t* self)
     assert(self);
 
     if (!self->state_file) {
-        log_warning("There is no state path set-up, can't store the state");
+        logDebug("There is no state path set-up, can't store the state");
```
Comment (ericclappier-eaton) [inline src/osrv.h]: Warning ?

File: src/osrv.h [hunk 9]
```diff
@@ -88,19 +88,19 @@ inline int s_osrv_load(s_osrv_t* self)
     assert(self);
 
     if (!self->state_file) {
-        log_warning("There is no state path set-up, can't load the state");
+        logDebug("There is no state path set-up, can't load the state");
```
Comment (ericclappier-eaton) [inline src/osrv.h]: Warning ?

**Review by ericclappier-eaton — APPROVED** [review]

**Review by ericclappier-eaton — CHANGES_REQUESTED** [review]

**Review by perrettecl — APPROVED** [review]

**Pull request merged**